terra-notebooks-playground/Py3 - How to add column descriptions.ipynb

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
Can the notebook be called "How to add BigQuery column descriptions"

Reply:
Done. Thanks!